src/core/Core.js

```diff
@@ -95,7 +95,7 @@ class Uppy {
       meta: Object.assign({}, this.opts.meta),
       info: {
         isHidden: true,
-        type: '',
+        type: 'info',
         msg: ''
```

Comment:
This property name was changed to `message` further down but not here

Reply:
👍 good catch, thx, fixed